.github/workflows/deploy-book.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy book
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 6 * * 1'  # Every Monday 6 AM UTC
```

Comment:
Why having a CRON, nothing is supposed to change if all versions are pinned.

Reply:
The real question is why not add cron